namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-prisoner-search-dev/resources/opensearch.tf

```diff
@@ -0,0 +1,74 @@
+module "hmpps_prisoner_search_opensearch" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-opensearch?ref=1.0.0"
+  application            = var.application
+  business-unit          = var.business_unit
+  eks_cluster_name       = var.eks_cluster_name
+  environment-name       = var.environment-name
+  infrastructure-support = var.infrastructure_support
+  is-production          = var.is_production
+  namespace              = var.namespace
+  team_name              = var.team_name
+  vpc_name               = var.vpc_name
+
+  engine_version = "OpenSearch_2.5"
+
+  cluster_config = {
+    instance_count = 2
+    instance_type  = "t3.small.search"
+  }
+
+  ebs_options = {
+    volume_size = 10
+  }
+}
+
+module "os_snapshots_s3_bucket" {
```

Comment:
presumably snapshots are supported for opensearch too 🤷.  Config copied from our existing elasticsearch instance.